inst/rmd/umap_qc.rmd

```diff
@@ -0,0 +1,101 @@
+## Dimensionality Reduction 
+
+The below plot shows the UMAP (Uniform Manifold Approximation and Projection) embeddings for each cell, coloring each cell by the total number of genes detected per cell.
+
+```{r message=FALSE}
+# create UMAP colored by number of genes detected 
+scater::plotUMAP(processed_sce,
+                 point_size = 0.3,
+                 point_alpha = 0.5,
+                 colour_by = "detected") +
+  scale_color_viridis_c()+
+  theme_bw() + 
+  guides(
+    color = guide_colorbar(title = "Number of \ngenes detected")
+  )
+```
+
+### Expression of highly variable genes
+
+The plots below show the same UMAP embeddings, coloring each cell by the expression level of the labeled gene.
+The genes chosen for plotting are the 12 most variable genes identified in the library.
+Gene symbols are used when available to label the UMAP plots.
+If gene symbols are not available, the Ensembl ID will be shown.
```

Comment:
Noting here that we are inconsistent about the use of `id` vs `ID`; we should probably clean that up. (We use "id" in sample metrics table, for example). I don't really care which we use, but we should try to be consistent. 

I only noticed this because it came up in a training document, and then I spent too long looking to figure out what is  "correct" and the answer is kind of neither, but probably "ID" is more common, mostly because people transferred ID as an initialism for "identity document" to an abbreviation for "identifier", even though they are not the same. But the training docs (mostly) use "id", because that seems to be what we have been doing...